src/azure-cli/azure/cli/command_modules/sql/_format.py

```diff
@@ -71,6 +71,33 @@ def _apply_format(result, format_group):
     # Apply format function to list
     return [format_group(item) for item in obj_list]
 
+###############################################
+#                sql instance-pool            #
+###############################################
+
+
+def instance_pool_table_format(result):
+    '''
+    Formats an instance pool or list of instance pools as summary results for display with "-o table".
+    '''
+    def _instance_pool_table_format(result):
+        '''
+        Formats an instance pool or list of instance pools as summary results for display with "-o table".
+        '''
+        from collections import OrderedDict
+        sku = result['sku']
+        return OrderedDict([
```

Comment:
I would not include ALL properties in the table. Just the most critical ones. For instance pool I suggest: name, rg, location, tier, family, capacity (see sql db & sql elastic pool for related examples that should be emulated).